docs/Tutorial_Top_Ten_3.ipynb

```diff
@@ -0,0 +1,251 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    idx_motifB = mp[:, 1][idx_motifA]
</pre><p>And then, we revise this as follows:</p><pre class="ql-syntax" spellcheck="false">nn_idx = mp[idx, 1]  # nn_idx : start index of 1st nearest neighbor of motif
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/676/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>